src/a-frame/component/tour.js

```diff
@@ -80,10 +80,28 @@ export default {
     this.animate(target)
   },
 
+  setCameraMode: function (mode) {
```

Comment:
I'm not 100% convinced yet, because this feels moving some relatively specific functionality (the editor's "camera mode") to a generic component.

But I guess it doesn't hurt, aye?